testing/tests/DevExpress.ui.widgets/drawer.tests.js

```diff
@@ -376,25 +376,44 @@ QUnit.module('Drawer behavior', () => {
         assert.ok($(instance._overlay.option('position').of).hasClass('dx-drawer-content'), 'target is ok');
     });
 
-    QUnit.test('show warning if deprecated \'target\' option is used', function(assert) {
-        sinon.spy(errors, 'log');
+    ['shrink', 'overlap', 'push'].forEach((openedStateMode) => {
+        QUnit.test(`warnings for deprecated 'target' option, ${openedStateMode}, target: notInitialized`, function(assert) {
+            sinon.spy(errors, 'log');
```

Comment:
Use the `stub` instead of `spy`. Keep console clean

Reply:
Done